content/2020/2020-10/2020-10-09-pulsar-weekly.md

```diff
@@ -0,0 +1,195 @@
+
+- [Topic Policy] Support null check to avoid the NPE for `internalSetTopicPolicies`.
+
+    https://github.com/apache/pulsar/pull/8201 (@[hangc0276](https://github.com/hangc0276))
+
+- [C++ Client] Add an epoch for C++ client `HandleBase` to handle the timeout issue when creating the producer.
+
+    https://github.com/apache/pulsar/pull/8191 (@[hangc0276](https://github.com/hangc0276))
+
+- [Dashboard] Fix the `consumerName` migration issue in the Django model of the Pulsar dashboard.
+
+    https://github.com/apache/pulsar/pull/8188 (@[sbourkeostk](https://github.com/sbourkeostk))
+
+- [Go Client] Fix the argument type of `pulsarProducerSendCallbackProxy`.
+
+    https://github.com/apache/pulsar/pull/8186 (@[massakam](https://github.com/massakam))
+
+- [Functions] Propagate user-defined parameters into instances of Pulsar Go functions.
+
+    https://github.com/apache/pulsar/pull/8132 (@[vaihtovirta](https://github.com/vaihtovirta))
+
+- [Bookie] Expose `ensemblePlacementPolicy` in `bookkeeper.conf`.
+
+    https://github.com/apache/pulsar/pull/8210 (@[hangc0276](https://github.com/hangc0276))
+
+- [Broker] Enable intercepting filters only when interceptors are configured.
+
+    https://github.com/apache/pulsar/pull/8157 (@[zymap](https://github.com/zymap))
+
+- [Python Client] Add default values for Python schema fields.
+
+    https://github.com/apache/pulsar/pull/8122 (@[hangc0276](https://github.com/hangc0276))
+
+### Notable Bug Fix
+
+- [C++ Client] Fix the unstable unit test of `testIndividualAck`.
+
+    https://github.com/apache/pulsar/pull/8141 (@[BewareMyPower](https://github.com/BewareMyPower))
+
+- [Client] Fix `ConsumerImpl` memory leaks.
+
+    https://github.com/apache/pulsar/pull/8160 (@[lhotari](https://github.com/lhotari))
+
+- [Broker] Fix error codes returned to the Pulsar client when the service unit is not ready.
+
+    https://github.com/apache/pulsar/pull/8147 (@[massakam](https://github.com/massakam))
+
+- [Schema] Fix the possible NPE when starting `BookkeeperSchemaStorage`.
+
+    https://github.com/apache/pulsar/pull/8172 (@[MaiCw4J](https://github.com/MaiCw4J))
+
+- [Admin] Fix the issue of failing to peek messages when the subscription name does not exist.
+
+    https://github.com/apache/pulsar/pull/8182 (@[hangc0276](https://github.com/hangc0276))
+
+- [Client] Fix the issue that the paused consumer receives new messages when reconnecting to the broker.
+
+    https://github.com/apache/pulsar/pull/8165 (@[massakam](https://github.com/massakam))
+
+- [Broker] Fix the exception that is thrown when closing the Pulsar service.
+
+    https://github.com/apache/pulsar/pull/8197 (@[hangc0276](https://github.com/hangc0276))
+
+- [Schema] Fix the issue that JSON schema deserializes bytes to strings.
+
+    https://github.com/apache/pulsar/pull/8140 (@[hangc0276](https://github.com/hangc0276))
+
+### Event / News
+
+- [TGIP] Weekly live stream about Pulsar and its ecosystem.
```

Comment:
shall we add this one:
ApacheCon @Home Draws Global Attendance

ApacheCon @Home took place from September 29th to October 1st. We hosted the Pulsar/BookKeeper track, including eighteen talks on [Pulsar/BookKeeper](https://www.apachecon.com/acah2020/tracks/pulsar.html). 
Presentations will be available on-demand soon, find out more at ApacheCon @Home.

Reply:
If the link for presentations/videos of ApacheCon is ready, we can add it. If just a sentence, we can hold it until the presentations or videos are ready. What do you think?